### DISCUSSION TITLE
FZF: how do i set prompt at bottom like telescope ?

### DISCUSSION THREAD

**Discussion by DrummyFloyd** [LazyVim/LazyVim#3595] [Q&A] [answered]
i"ve searched in fzf doc, but did not find ( i'm maybe dumb ^^and pretty sure about it ahah )

can someone, can help me ?

**Comment by folke** [comment]
Check the fzf-lua docs. You're looking for fzf_opts

**Comment by rubiin** [comment, marked as answer]
@DrummyFloyd this works
```lua
  {
    "ibhagwan/fzf-lua",
    opts = {
      fzf_opts = { ["--layout"] = "default" },
    },
  }
```

![image](https://github.com/LazyVim/LazyVim/assets/8222059/8f499769-9fdc-4102-bffc-bb7e470aafcb)

**Reply by DrummyFloyd** [reply]
thank you =)

**Reply by rubiin** [reply]
Glad it works for you :)
